pokemongo_bot/logger.py

```diff
@@ -18,3 +17,106 @@ def log(msg, color=None):
     logger = logging.getLogger('generic')
     logger.info(msg)
     warnings.warn(message, DeprecationWarning)
+
+class LcdHandler(logging.Handler):
```

Comment:
The LcdHandler is beyond the objective of this PR and being reimplemented by @brantje. Can you remove it, please?